docs/packages/pkg/PNG.rst

```diff
@@ -18,9 +18,10 @@ PNG
   :start-after: # DOCUMENTATION_START {
   :end-before: # DOCUMENTATION_END }
 
-For compatibility with
+For compatibility with the
 `FindPNG <https://cmake.org/cmake/help/latest/module/FindPNG.html>`__
-module, the ``ALIAS`` target ``PNG::PNG`` can be used too:
+module, the ``ALIAS`` target ``PNG::PNG`` can be used too (requires CMake
```

Comment:
That's not applied for the current state. Please submit this change with a new PNG version.

Update instructions:
- https://docs.hunter.sh/en/latest/creating-new/update.html

Reply:
Right, I added the latest 1.6.26 and 1.6.36 packages and updated the default version to the latest 1.6.26 package.